E-legal-iOS/elegal-ios
# Lawnet and Cases screen design

#### What's this PR do?

This PR implements the design for Lawnet and Cases screens.
#### Reviewers

@ahmad-ishfaq  @sweetevil  
#### How should this be manually tested?

Go to this branch and run the application.
Go to Create account screen and click on create account button, you'll move to Home screen and then click on left menu button on second button "Forum" you can see the Lawnet screen design and after that if you click on "Case"  button you'll the screen designed for Cases.
![simulator screen shot 26-jul-2016 9 19 16 pm](https://cloud.githubusercontent.com/assets/10810397/17150490/1f9718bc-5389-11e6-9630-fa30b7f8a136.png)
![simulator screen shot 26-jul-2016 9 19 14 pm](https://cloud.githubusercontent.com/assets/10810397/17150491/1fa09b30-5389-11e6-8457-547da8621857.png)
![simulator screen shot 26-jul-2016 9 18 42 pm](https://cloud.githubusercontent.com/assets/10810397/17150493/1fa53712-5389-11e6-8ee5-e25734c43838.png)
![simulator screen shot 26-jul-2016 9 18 40 pm](https://cloud.githubusercontent.com/assets/10810397/17150492/1fa303b6-5389-11e6-8157-b293bf8b46f4.png)
![simulator screen shot 26-jul-2016 9 17 58 pm](https://cloud.githubusercontent.com/assets/10810397/17150494/1fcc926c-5389-11e6-937e-a9879da875d7.png)
![simulator screen shot 26-jul-2016 9 17 53 pm](https://cloud.githubusercontent.com/assets/10810397/17150495/1fd70abc-5389-11e6-902e-b2ff3414f0a3.png)
